doc/source/api/api_transform.rst

```diff
@@ -0,0 +1,17 @@
+=========
+Transform
+=========
+
+Functions to help transforming of data between formats, regridding etc.
```

Comment:
"Functions that can convert from underlying databases or model outputs into the standardised OpenGHG format. This could include, for example, creating a Flux file for a limited domain based on data from the EDGAR database. In constrast to standardisation functions, this will usually include some amount of transformation such as selection and/or regridding."

Reply:
Thanks!